src/neuroconv/tools/audio/audio.py

```diff
@@ -0,0 +1,85 @@
+from typing import Optional
+from warnings import warn
+
+from hdmf.backends.hdf5 import H5DataIO
+from ndx_sound import AcousticWaveformSeries
+from pynwb import NWBFile
+
+from neuroconv.tools.hdmf import SliceableDataChunkIterator
+from neuroconv.utils import ArrayType
+
+
+def add_acoustic_waveform_series(
+    acoustic_series: ArrayType,
+    rate: float,
```

Comment:
@CodyCBakerPhD 
Yes, my reasoning behind was to separate the audio reader part from this method (they usually return the audio data and the sampling rate from the file), but you're right, instead of this it could take like an `Audio` object, where we'd expect that it has a `data` and `rate` attribute. What do you think about this?